src/providers/push/fcm/fcm.provider.ts

```diff
@@ -94,11 +96,9 @@ export class FcmProvider implements Provider<FcmNotification> {
           data: {...message.options.data},
         };
 
-        promises.push(
-          this.fcmService
-            .messaging()
-            .send(msgToTransfer, (message.options.dryRun = false)),
-        );
+        message.options.dryRun = false;
```

Comment:
what is this dryrun for ?

Reply:
This was done to resolve the sonar issue, where I extracted the assignment of message.options.dryRun

Reply:
why not do this  message.options.dryRun ?? false.
as suggested above